include/avif/internal.h

```diff
@@ -262,14 +262,14 @@ struct avifCodecInternal;
 
 typedef enum avifEncoderChange
 {
-    AVIF_ENCODER_CHANGE_MIN_QUANTIZER = (1 << 0),
-    AVIF_ENCODER_CHANGE_MAX_QUANTIZER = (1 << 1),
-    AVIF_ENCODER_CHANGE_MIN_QUANTIZER_ALPHA = (1 << 2),
-    AVIF_ENCODER_CHANGE_MAX_QUANTIZER_ALPHA = (1 << 3),
-    AVIF_ENCODER_CHANGE_TILE_ROWS_LOG2 = (1 << 4),
-    AVIF_ENCODER_CHANGE_TILE_COLS_LOG2 = (1 << 5),
-
-    AVIF_ENCODER_CHANGE_CODEC_SPECIFIC = (1 << 31)
```

Comment:
Google's internal ClangTidy check warns that `1 << 31` shifts the 1 to the sign bit of a signed int.